runtime/near-evm-runner/src/near_ext.rs

```diff
@@ -64,6 +64,10 @@ impl<'a> NearExt<'a> {
 }
 
 impl<'a> vm::Ext for NearExt<'a> {
+    /// EIP-1344: Returns the current chain's EIP-155 unique identifier.
+    /// See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1344.md
+    fn chain_id(&self) -> u64 { todo!() }
```

Comment:
I think we have `chain_id` in genesis config, can you pass it to `NearExt`?

Reply:
Yes, working on it

Reply:
Implemented in baeb9d9baab8bdc83b50de16c364a7b2b71cbc59.

The chain ID enters near-evm-runner at:

https://github.com/near/nearcore/blob/baeb9d9baab8bdc83b50de16c364a7b2b71cbc59/runtime/runtime/src/actions.rs#L56

https://github.com/near/nearcore/blob/baeb9d9baab8bdc83b50de16c364a7b2b71cbc59/runtime/near-evm-runner/src/lib.rs#L559

From there, it must be passed through to everywhere where it is needed.